src/pages/PrimeCalculator/Form/validateNumericString.ts

```diff
@@ -0,0 +1,12 @@
+import BigNumber from 'bignumber.js';
+
+export const validateNumericString = (str: string, minValue = new BigNumber(0)) => {
+  if (!str) {
+    return true;
```

Comment:
This doesn't look right. false, emptyStrings, undefined, and null are not numeric strings